source/install.txt

```diff
@@ -36,11 +36,15 @@ Select your operating system:
          - 64-bit version of Microsoft Windows 7 or later.
          - MongoDB 3.6 or later.
          - `Microsoft .NET Framework version 4.5 or later <https://www.microsoft.com/en-us/download/details.aspx?id=30653>`__.
-
+         
            The |compass-short| installer prompts you to install the
            minimum required version of the .NET framework if it is not
            already installed on your system.
 
+         - Starting the installation as an administrator in your Azure
```

Comment:
[question]

This makes it sound like I _must_ use Azure VDI or PowerShell to complete the installation, which isn't the case. I was just able to complete the installation on my Windows computer and did not need to use PowerShell.

It sounds like the ticket is suggesting that we update the docs to say that you should only install using Azure VDI or PowerShell, but I worry this may be an unnecessary complication for the majority of users.

I'm wondering if this be reworded to something like:

"Starting the installation as an administrator if you are installing using Azure Virtual Desktop Infrastructure (VDI) or Microsoft PowerShell."

Reply:
Makes sense! Will update, Jeff!

Reply:
@jeff-allen-mongo please take another look? I made the changes.